container.fc

```diff
@@ -62,6 +64,7 @@
 /var/lib/docker/overlay(/.*)?	gen_context(system_u:object_r:container_ro_file_t,s0)
 /var/lib/docker/overlay2(/.*)?	gen_context(system_u:object_r:container_ro_file_t,s0)
 
+/var/lib/nerdctl(/.*)?	gen_context(system_u:object_r:container_var_lib_t,s0)
```

Comment:
If /var/lib/nerdctl content needs to be read within containers without a relabel, IE they are just volume mounted int, then it might be better to  label as container_ro_file_t. Containers running with container_t are not able to read content labeled container_var_lib_t.